src/workloads/scale/ContentionTTLDeletions.yml

```diff
@@ -51,7 +39,7 @@ Actors:
   Phases:
     OnlyActiveInPhases:
       Active: [1]
```

Comment:
Do you want these admin commands to run before the data is loaded? Currently they are both set to run in Phase 1, so they will run in parallel.

Reply:
Sounds good!